com.onepassword.OnePassword.yml

```
@@ -0,0 +1,49 @@
+app-id: com.1password.onepassword
```

```suggestion
app-id: com._1password.OnePassword
```

you can't have a number starting.

This need to be changed everywhere else. and the file names need to match.

---

`onepassword.com` redirects to `1password.com`. Can't `com.onepassword` be used instead of `com._1password`?

---

as the time I check it wasn't working. could have been on my end.

Now given the further development that there is a flatpak, the same ID must be used.